steps/reasons-for-appealing/evidence-upload/fileTypeWhitelist.js

```diff
@@ -21,12 +22,15 @@ module.exports = [
   '.txt',
   '.doc',
   '.docx',
+  '.rtf',
```

Comment:
does this get converted to PDF and if so does the PDF converteter accept RTFs? Assume there's already handling for this in Tribunals?

Reply:
Yes, recently Chris has implemented RTF to PDF converter in tribunals api